crates/agent/src/context_strip.rs

```diff
@@ -98,11 +99,23 @@ impl ContextStrip {
                 .as_ref()
                 .and_then(|thread_store| thread_store.upgrade())
                 .and_then(|thread_store| thread_store.read(cx).prompt_store().as_ref());
+
+            let current_model = LanguageModelRegistry::global(cx)
```

Comment:
This will not work reliably, because the context strip is also used in the inline assistant. Technically the inline assistant can use a different model (via `inline_assistant_model`). To fix this we need to store in which context we are operating in (Thread/Inline Assistant). Once #32076 is merged we can pass `ModelUsageContext::Thread`/`ModelUsageContext::InlineAssistant` into the context strip.